main.py

```diff
@@ -16,7 +15,15 @@ def login():
         
     if 'name' not in data or 'password' not in data:
         return 'Name or/and password are missing.', 400
-    
+
+    if not isinstance(data['name'], str):
+        data['name'] = str(data['name'])
```

Comment:
We talked about the user sending us malformed data, why would we want to cast it to string instead of erroring out?

Reply:
Change to erroring out.